tests/test_config_parser.py

```diff
@@ -36,6 +36,13 @@ def case_pdb(sarg=None):
     parser.get_parsed_content()
 
 
+@TimedCall(seconds=100, force_quit=True)
+def case_pdb_inst(sarg=None):
```

Comment:
I am not very familiar with pdb test, do we need to check / assert anything in this function?

Thanks.